opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/dto/result/body/AccountListBody.java

```diff
@@ -1,10 +1,17 @@
 package de.adorsys.opba.protocol.api.dto.result.body;
 
+import lombok.Builder;
 import lombok.Data;
 
+import java.util.List;
+
+@Builder
```

Comment:
If you use Builder, then make the class immutable and drop Data annotation

Reply:
ok